tests/unit/test_refresh_token_authorizer.py

```diff
@@ -16,188 +14,58 @@ def setUp(self):
-
-    def test_check_expiration_time_no_expiration(self):
-        """
-        Confirms a new access_token is gotten if expires_at is set to None
-        """
-        self.authorizer.expires_at = None
-        self.authorizer._check_expiration_time()
-        self.assertEqual(self.authorizer.access_token,
-                         self.rs_data['access_token'])
-        self.assertEqual(self.authorizer.expires_at,
-                         self.rs_data['expires_at_seconds'] -
-                         EXPIRES_ADJUST_SECONDS)
-
-    def test_set_authorization_header(self):
-        """
-        Sets authorization header on a test dictionary, confirms expected value
-        """
-        header_dict = {}
-        self.authorizer.set_authorization_header(header_dict)
-        # confirm value
-        self.assertEqual(header_dict["Authorization"],
-                         "Bearer " + self.access_token)
-
-    def test_set_authorization_header_existing(self):
-        """
-        Confirms that an existing Authorization field is overwritten
-        """
-        header_dict = {"Header": "value",
-                       "Authorization": "previous_value"}
-        self.authorizer.set_authorization_header(header_dict)
-        # confirm values
-        self.assertEqual(header_dict["Authorization"],
-                         "Bearer " + self.access_token)
-        self.assertEqual(header_dict["Header"], "value")
-
-    def test_set_authorization_header_expired(self):
-        """
-        Waits for the access_token to expire, then sets authorization header
-        Confirms header value uses the new access_token.
-        """
-        header_dict = {}
-        time.sleep(1)
-
-        self.authorizer.set_authorization_header(header_dict)
-        # confirm value
-        self.assertEqual(header_dict["Authorization"],
-                         "Bearer " + self.rs_data['access_token'])
-
-    def test_set_authorization_header_no_token(self):
-        """
-        Sets the access_token to None, then sets authorization header
-        Confirms header value uses the new access_token.
-        """
-        header_dict = {}
-        self.authorizer.access_token = None
-
-        self.authorizer.set_authorization_header(header_dict)
-        # confirm value
-        self.assertEqual(header_dict["Authorization"],
-                         "Bearer " + self.rs_data['access_token'])
-
-    def test_set_authorization_header_no_expires(self):
-        """
-        Sets expires_at to None, then sets authorization header
-        Confirms header value uses the new access_token.
-        """
-        header_dict = {}
-        self.authorizer.expires_at = None
-
-        self.authorizer.set_authorization_header(header_dict)
-        # confirm value
-        self.assertEqual(header_dict["Authorization"],
-                         "Bearer " + self.rs_data['access_token'])
 
-    def test_handle_missing_authorization(self):
+    def test_multiple_resource_servers(self):
```

Comment:
I definitely should have added this test when changing this behavior.
Thanks for having my back! :+1: